src/ui/DApps/IslandView.tsx

```diff
@@ -16,14 +16,15 @@ import {
 } from "redux-state"
 import FullPageLoader from "shared/components/FullPageLoader"
 import { Route, Switch } from "react-router-dom"
-import { useDisplayedPopulation } from "shared/hooks"
+import { useDisplayedPopulation, useReflect } from "shared/hooks"
 
 export default function IslandView() {
   const islandMode = useDappSelector(selectIslandMode)
   const hasLoadedRealmData = useDappSelector(selectHasLoadedRealmData)
   const hasLoadedSeasonInfo = useDappSelector(selectHasLoadedSeasonInfo)
   const hasBalances = useDappSelector(selectHasLoadedBalances)
 
+  useReflect()
```

Comment:
I wonder if we shouldn't just move the `useReflect` into the `IslandPresence` to encapsulate the reflect feature more 🤔